### DISCUSSION TITLE
Consider securing the Base part with a through-bolt

### DISCUSSION THREAD

**Discussion by dflemstr** [stevenbennett/DIYson#25] [Ideas]
The Base part currently consists of a short bolt to attach it to an aluminum extrusion, and then the 40mm spacer to attach it to the Desk Mount.  This makes it quite brittle.

Gonna try to visualize using ASCII art, the drawing is not to scale :)

```
                                                      Bolt

                            ┌──────────────────────────┐ ┌───────────────────────────────┐
                            │                          │ │                               │
┌───────────────────────────┼───────────────┐   ┌───┐  │ │                               │
│                           │               │   │   ├──┼─┼─────┐                         │
│                           │               │   │   │  │ │     │                         │
│                           │               │   │   ├──┼─┼─────┘                         │
└───────────────────────────┼───────────────┘   └───┘  │ │                               │
                            │                          │ │                               │
40mm spacer                 └──────────────────────────┘ └───────────────────────────────┘
                                   3D printed base               Aluminum extrusion
```

An alternative is to use a long bolt that goes through both the 40mm spacer and through the Base 3D printed part and all the way into the aluminum extrusion.

```
Bolt                   ┌──────────────────────────┐   ┌─────────────────────────┐
                       │                          │   │                         │
┌───┐  ┌───────────────┼────────────────────────┐ │   │                         │
│   ├──┼───────────────┼────────────────────────┼─┼───┼────────┐                │
│   │  │               │                        │ │   │        │                │
│   ├──┼───────────────┼────────────────────────┼─┼───┼────────┘                │
└───┘  └───────────────┼────────────────────────┘ │   │                         │
                       │                          │   │                         │
                       └──────────────────────────┘   └─────────────────────────┘
        40mm spacer           3D printed base                 Aluminum extrusion
```

I've managed to do this with a 55mm or 60mm bolt quite easily with some minor tweaks to the 3D model using slicer hacks, so I think it should be quite straight-forward to update the original model as well.

**Comment by stevenbennett** [comment]
@dflemstr Cool idea, this was one of my initial concepts actually (though I never mentioned it in any build logs). Since you have this implemented already maybe you can answer a question for me; does the screw ever unseat itself when turning the lamp counter-clockwise? Early on I worried that the friction on the screw head and a counter-clockwise turn might be enough to unscrew it, but perhaps that weight of the lamp prevents this. Any findings here?

Also thanks for the art, I like these :)

**Reply by dflemstr** [reply]
I have implemented this and so far it is working fine!  I could've added some thread locker but so far I didn't have to. I also even skipped the 3D printed base all together and just used the 40mm spacer with the same end cap that's used for the horizontal beam, then bolting the 40mm spacer through that one to the underside of the extrusion; this turned out to add some extra stability which meant that's the solution I'm currently still using.

Here's some more ASCII art to clarify what I mean, now with arrows!

```
                             ┌─┬─────────────────────┐
                             │ │                     │
┌──┐ ┌─────────────────────┐ │ │                     │
│  ├─┼─────────────────────┼─┼─┼─────────────┐       │
│  │ │                     │ │ │             │       │
│  ├─┼─────────────────────┼─┼─┼─────────────┘       │
└──┘ └─────────────────────┘ │ │                     │
 ▲                           │ │                     │
 │      ▲                    └─┴─────────────────────┘
 │      │                     ▲      ▲
 │      40mm spacer           │      └─Extrusion
 │                            │
 Long bolt                    End Cap (optional).stl
```

**Reply by dflemstr** [reply]
I've also managed to connect this to a normal microphone stand desk mount, skipping the 3D printed base + C clamp, which is an additional bonus

**Reply by stevenbennett** [reply]
Awesome, thanks for reporting back on this. I will give it a try and consider making it an official improvement :)

**Comment by stevenbennett** [comment]
@dflemstr Oh sorry just reread and realized you may not have actually implemented this yourself. Maybe somebody can give it a try

**Comment by legopc** [comment]
The head of an M5 bolt is a little smaller then then washer so if you leave some space around the head of the bolt it should not be an issue. By the way, would it be possible to get the Fusion 360 files for the base and related components? I'm completely new to Fusion360 and 3d printing so my printer is not yet as accurate as I thing yours is so the spacer for example doesnt fit in the base.

**Reply by dflemstr** [reply]
+1, would love to be able to try out some stuff (both 6mm belt and improving this through-bolt version of the 3D printed base)

**Reply by stevenbennett** [reply]
I'll upload the base Fusion 360 files for you guys

**Comment by legopc** [comment]
I'm testing this with the original base and a longer screw with a couple of the washers that are in the BOM, and I can definitely recommend this. It makes the whole lamp sturdier and nicer to use overall. One thing im running in to though is because I dont have the original design files, the tolerances on the hole in the base are just to tight when I screw in the bolt enough to the point that it doesnt undo itself easily, it crushes the washer I guess to the point of it not fitting "freely" anymore. I am currently having to redesign the base for myself which is unfortunate because printed in PETG with 25% infill it has been quite strong and this modification also eliminates the need for an extra 3d printed base by just screwing into the base.

**Reply by stevenbennett** [reply]
@legopc (@dflemstr too) I should have posted here but I uploaded the base CAD file for you: https://github.com/stevenbennett/DIYson/blob/main/DIYson%20Express/Printed%20Parts/F3D/Base.f3d

Thanks for contributing this improvement :) I will test it myself soon.

**Reply by legopc** [reply]
Ah yes I saw those files but I meant the desk mount base. The extrusion end base you linked can completely be skipped with the bolt washer variant of mounting it. But like you said yourself in one of your video's, I'm also currently looking for a nice slip ring, so all this work might be for nothing.

**Reply by stevenbennett** [reply]
Haha, it's my fault for not commenting here. Have fun :)

**Reply by legopc** [reply]
Ah you saw before I could edit haha. I assumed it was the right base until I downloaded it to check and well see the edit above haha. @stevenbennett

**Reply by stevenbennett** [reply]
@legopc Whoops, totally missed that. Just uploaded the desk mount: commit 178f66f

**Comment by legopc** [comment]
Thanks for the designs @stevenbennett. I changed the diameter to 10.3 (the calibration could be tighter), and it fits really nicely. This is honestly a great upgrade to the lamp. I also printed the base in 75% infill in ABS+ and I can really put some pressure without breaking it and the tight fit really makes the whole lamp nice to use, this is definitely an upgrade that needs to be incorporated. It almost completely Z wobble and makes almost every movement with the lamp more crisp.
